src/Rules/DateBetween.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Spatie\ValidationRules\Rules;
+
+use Carbon\Carbon;
+use Illuminate\Contracts\Validation\Rule;
+
+class DateBetween implements Rule
+{
+    /** @var \Carbon\Carbon */
+    protected $start;
+
+    /** @var \Carbon\Carbon */
+    protected $end;
+
+    /** @var bool */
+    protected $orEquals = false;
```

Comment:
Let's make this 'true` by default. I'm thinking that will be the most wanted behaviour.